### PR TITLE
Fix the port for when we use the HTTPS protocol.

### PR DESCRIPTION
**Pull request by dompuiu**
## Description

Sandbox was not able to make the API calls when you use the HTTPS version.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -12,7 +12,9 @@
 
     <script>
       const protocol = window.location.protocol;
-      window.EXPRESS_PUBLIC_URL = `${protocol}//localhost:3000`;
+      const port = protocol === 'https:' ? 4000 : 3000;
+
+      window.EXPRESS_PUBLIC_URL = `${protocol}//localhost:${port}`;
       window.REACT_PUBLIC_URL = `${protocol}//localhost:3001`;
```
Comment (brenthosie) [inline public/index.html:18]: @dompuiu does this need to be updated as well?

Comment (dompuiu) [reply public/index.html:18]: @brenthosie I am not sure if the create-react-app offers an option to run the dev server on 2 different ports at the same time. I am not even sure if we use this variable anywhere. I am not able to find it.

Comment (brenthosie) [reply public/index.html:18]: if it's not there, let's pull it out on this PR.

Comment (dompuiu) [reply public/index.html]: I've removed it.

**Pull request merged**